Formula/gitversion.rb

```diff
@@ -1,27 +1,31 @@
 class Gitversion < Formula
   desc "Easy semantic versioning for projects using Git"
-  homepage "https://github.com/GitTools/GitVersion"
-  url "https://github.com/GitTools/GitVersion/releases/download/5.0.1/GitVersion-bin-fullfx-v5.0.1.zip"
-  sha256 "9b543d3e42e0d5e6fab0b44553cb6bbbb0e31431030ef761fc1a50c845fd166a"
+  homepage "https://gitversion.net"
+  url "https://github.com/GitTools/GitVersion/archive/5.3.7.tar.gz"
+  sha256 "70c6f867e4a85325ba1e54211fd014d9b3ec9be43bb828393e1f0d6a5e33cefb"
   license "MIT"
 
-  bottle :unneeded
+  depends_on "dotnet"
 
-  depends_on "mono"
+  def install
+    system "dotnet", "build",
+           "--configuration", "Release",
+           "--framework", "netcoreapp3.1",
+           "--output", "out",
+           "src/GitVersionExe/GitVersionExe.csproj"
 
-  uses_from_macos "icu4c"
+    libexec.install Dir["out/*"]
 
-  def install
-    libexec.install Dir["*"]
     (bin/"gitversion").write <<~EOS
       #!/bin/sh
-      exec "#{Formula["mono"].opt_bin}/mono" "#{libexec}/GitVersion.exe" "$@"
+      exec dotnet "#{libexec}/gitversion.dll" "$@"
```

Comment:
```suggestion
      exec "#{Formula["dotnet"].opt_bin}/dotnet" "#{libexec}/gitversion.dll" "$@"
```

Reply:
Why should the `system` call in `install` expect `dotnet` to exist on `PATH` while the one in the executable script should not?

Reply:
Because the first one is ran within the formula install, where we control the path. Here you're writing a file that users will call and obviously we don't control their path.

Reply:
Makes sense, thanks.